src/js/containers/covid19/SummaryInsightsContainer.jsx

```diff
@@ -43,53 +45,71 @@ const SummaryInsightsContainer = ({
     const [numberOfAwards, setNumberOfAwards] = useState(null);
     const initialInFlightState = overviewData.map((d) => d.type);
     const [inFlightList, , removeFromInFlight, resetInFlight] = useInFlightList(initialInFlightState);
-    const defCodes = useSelector((state) => state.covid19.defCodes);
+    const { defCodes, allAwardTypeTotals } = useSelector((state) => state.covid19, isEqual);
 
     useEffect(() => {
+        if (assistanceOnly && (activeTab === 'all' && prevTab === 'all')) {
```

Comment:
This reminds me of using component lifecycle methods. W/ useEffect, we have to use a different mental model. I'm not confident this prevTab value is going to be reliable.

Also, are we making duplicative requests to this endpoint in production?  If so, that could be a bug you can scope outside this delivery.

Reply:
As mentioned in our quick call, this particular duplicate call was noticed as a result of "all" for CFDA being considered something different than the other tables. Want to make sure we aren't sending a second request for CFDA when an update to `allAwardTypeTotals` triggers this hook.